Client/Frontend/Fakespot/FakespotCoordinator.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+protocol FakespotCoordinatorDelegate: AnyObject {
+    // Define any coordinator delegate methods here
+}
+
+class FakespotCoordinator: BaseCoordinator {
+    weak var parentCoordinator: FakespotCoordinatorDelegate?
+    weak var delegate: FakespotCoordinatorDelegate?
+
+    let fakespotViewController: FakespotViewController
+    let viewModel: FakespotViewModel
```

Comment:
Can be private? And I think unless I am missing something, the view model doesn't need to be kept here since it's passed and kept as a reference in the VC.

Also, wondering - so far we've kept reference to VC in the coordinators. I am wondering if we could avoid that in any way. What do you think?

Reply:
I remember for BrowserViewController there were already some navigation methods that were handled there and initially we were forwarding those routes, that's why we needed that reference.

And I agree, view controller will communicate with view model for business logic, and navigation will be forwarded to the coordinator through the delegate.